server/gokbg3/src/integration-test/groovy/gokbg3/IntegrationControllerSpec.groovy

```diff
@@ -45,586 +44,605 @@ class IntegrationControllerSpec extends Specification {
   }
 
   def cleanup() {
-    if (last) {
-      CuratoryGroup.findByName('TestGroup1')?.refresh().expunge()
-      Org.findByName("American Chemical Society")?.refresh().expunge()
-    }
+    if (CuratoryGroup.findByName('TestGroup1'))
```

Comment:
Cleanup is run after each test, but we need these objects for multiple test so we only delete them after running all tests.

Reply:
This breaks the general contract that tests shall be atomic and do not change the state of the system after being run. Furthermore, the order of tests is not defined (eg. parallel test runs) and therefore unsure wether the last test is run after all others.
To put it short: I'd rather not to...

Reply:
I see your point. In that case we would need to create those two objects in the `setup()`, and adjust the tests for assertOrg and assertGroup to create their own (and different) objects, which we will have to cleanup as well.